src/ui/blueprints/blog-post/PageBlueprintEditor.tsx

```diff
@@ -0,0 +1,101 @@
+import { ReactElement, useEffect, useState } from 'react';
+import { FieldEditor } from '@/ui/blueprints/FieldEditor';
+import { Field } from '@/model/field/Field';
+import { Page } from '@/model/subject/Page';
+import { PageBlueprint } from '@/model/blueprint/Page';
+import { CommandTypes, sendCommandToContent } from '@/bus/Command';
+import { EventTypes } from '@/bus/Event';
+import { ContentEventHandler } from '@/ui/blueprints/ContentEventHandler';
+
+interface Props {
+	blueprint: PageBlueprint;
+	subject: Page;
+	onFieldChanged: ( name: string, field: Field, selector: string ) => void;
+}
+
+export function PageBlueprintEditor( props: Props ) {
```

Comment:
Right now this file is under the `/ui/blueprints/blog-post/` directory which was intended just for blog posts, so I think we should either move this file to `/ui/blueprints/page/` or, given that there's in currently only one file per sub directory, move them both to `/ui/blueprints/`.